src/Adapter/Currency/CommandHandler/EditCurrencyHandler.php

```diff
@@ -268,6 +268,7 @@ private function updateNameAndSymbol(Currency $entity, $newIsoCode)
                 $langIds = Language::getLanguages(true, false, true);
                 $entity->name = [];
                 $entity->symbol = [];
+                $entity->numeric_iso_code = $currency->getNumericIsoCode();
```

Comment:
You cast it to `(int)` in `AddCurrencyHandler`, why not casting it here as well?